pom.xml

```diff
@@ -5,7 +5,7 @@
 	<parent>
 		<groupId>org.springframework.boot</groupId>
 		<artifactId>spring-boot-starter-parent</artifactId>
-		<version>3.1.3</version>
+		<version>3.3.3</version>
```

Comment:
Maven build fails with 3.3.3. The latest available is 3.2.2 - can you please update to that